src/docfx/restore/Restore.cs

```diff
@@ -3,14 +3,18 @@
 
 using System;
 using System.Collections.Concurrent;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
+using System.Linq;
 using System.Threading.Tasks;
 
 namespace Microsoft.Docs.Build
 {
     internal static class Restore
     {
+        private const int MaxWorkTreeCount = 5;
```

Comment:
>MaxWorkTreeCount [](start = 26, length = 16)

make this configurable using environment variable?

Reply:
I don't think the end user need to know it, let's just hardcode it and make it configurable when we want to implement `docfx gc` function?

---
In reply to: [201593922](https://github.com/dotnet/docfx/pull/3042#discussion_r201593922) [](ancestors = 201593922)

Reply:
ok

---
In reply to: [201595705](https://github.com/dotnet/docfx/pull/3042#discussion_r201595705) [](ancestors = 201595705,201593922)